test/FunctionalTests/MvcTests/RazorEmbeddedResource.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+using Xunit;
+
+namespace EntropyTests
+{
+    public static class RazorEmbeddedResource
```

Comment:
The method was made public and it's referencing this type to locate the project, but nothing apart from that.